lapack/testlapack/dlangb_bench.go

```diff
@@ -0,0 +1,51 @@
+// Copyright ©2020 The Gonum Authors. All rights reserved.
```

Comment:
I missed this. We're in 2021.

Reply:
Thanks, a leftover from copying `dlantb_bench.go`.